qa/tasks/mgr/dashboard/test_cephfs.py

```diff
@@ -32,19 +32,30 @@ def test_cephfs_data(self):
     @authenticate
     def test_cephfs_mds_counters(self):
         fs_id = self.fs.get_namespace_id()
-        data = self._get("/api/cephfs/mds_counters/{}".format(fs_id))
+        data = self._get("/api/cephfs/{}/mds_counters".format(fs_id))
         self.assertStatus(200)
 
         self.assertIsInstance(data, dict)
         self.assertIsNotNone(data)
 
     @authenticate
     def test_cephfs_mds_counters_wrong(self):
-        data = self._get("/api/cephfs/mds_counters/baadbaad")
+        self._get("/api/cephfs/mds_counters/baadbaad")
         self.assertStatus(400)
```

Comment:
it now returns 404. see http://pulpito.ceph.com/kchai-2018-05-31_12:14:16-rados-wip-kefu-testing-2018-05-31-1806-distro-basic-smithi/2614165/

```
2018-05-31T14:29:25.215 INFO:tasks.cephfs_test_runner:======================================================================
2018-05-31T14:29:25.215 INFO:tasks.cephfs_test_runner:FAIL: test_cephfs_mds_counters_wrong (tasks.mgr.dashboard.test_cephfs.CephfsTest)
2018-05-31T14:29:25.215 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2018-05-31T14:29:25.216 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2018-05-31T14:29:25.216 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2018-05-31-1806/qa/tasks/mgr/dashboard/helper.py", line 25,
in decorate
2018-05-31T14:29:25.216 INFO:tasks.cephfs_test_runner:    return func(self, *args, **kwargs)
2018-05-31T14:29:25.216 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2018-05-31-1806/qa/tasks/mgr/dashboard/test_cephfs.py", line
 44, in test_cephfs_mds_counters_wrong
2018-05-31T14:29:25.216 INFO:tasks.cephfs_test_runner:    self.assertStatus(400)
2018-05-31T14:29:25.216 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2018-05-31-1806/qa/tasks/mgr/dashboard/helper.py", line 246,
 in assertStatus
2018-05-31T14:29:25.216 INFO:tasks.cephfs_test_runner:    self.assertEqual(self._resp.status_code, status)
2018-05-31T14:29:25.216 INFO:tasks.cephfs_test_runner:AssertionError: 404 != 400
```

Reply:
It's fixed now.